Tests/test_cellosaurus.py

```diff
@@ -90,13 +94,18 @@ def test_parse(self):
             self.assertEqual(record["RX"][0], "PubMed=566722;")
             self.assertEqual(record["RX"][1], "PubMed=19630270;")
             self.assertEqual(record["RX"][2], "PubMed=20143388;")
-            self.assertEqual(record["WW"][0], "http://iclac.org/"
-                                              "wp-content/uploads/"
-                                              "Cross-Contaminations-v7_2.pdf")
-            self.assertEqual(record["CC"][0],
-                             "Problematic cell line: Contaminated. "
-                             "Shown to be a HeLa derivative "
-                             "(PubMed 566722, PubMed 20143388).")
+            self.assertEqual(
+                record["WW"][0],
+                "http://iclac.org/"
+                "wp-content/uploads/"
+                "Cross-Contaminations-v7_2.pdf",
```

Comment:
Will that URL fit on one line now (within 88 char limit)?

Reply:
yes, done